staging/src/k8s.io/apimachinery/pkg/api/errors/errors.go

```diff
@@ -184,6 +185,30 @@ func NewConflict(qualifiedResource schema.GroupResource, name string, err error)
 	}}
 }
 
+// NewApplyConflict returns an error including details on the requests apply conflicts
+func NewApplyConflict(conflicts merge.Conflicts, err error) *StatusError {
```

Comment:
`conflicts` and `err` are the exact same variable with one casted to the type of the other, that's surprising that you take both as parameters.